image_provisioner/playbooks/roles/atomic-kickstart/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
This role is called `atomic-kickstart` however it's pretty much exclusively building pbench containers. Please name appropriately.